project-base/storefront/components/Pages/StoreDetail/StoreDetailContent.tsx

```
@@ -25,100 +25,93 @@ export const StoreDetailContent: FC<StoreDetailContentProps> = ({ store }) => {
+                                {store.country.name}
+                            </p>
+                        </StoreSection>
 
-                    {store.phone || store.email ? (
-                        <InfoItem>
-                            <StoreContact email={store.email} phone={store.phone} />
-                        </InfoItem>
-                    ) : null}
+                        {!!store.directions && (
+                            <StoreSection title={t('How to reach us')}>
+                                <p dangerouslySetInnerHTML={{ __html: store.directions }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)